.github/workflows/python-package.yml

```diff
@@ -9,6 +9,9 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+  schedule:
+    # This job runs at 00:00 JST on Friday.
+    - cron: '0 0 * * *'
```

Comment:
[must]
I believe Github Actions timezone is UTC, and you need to add 9 hours to make it JST.
Something like `0 9 * * 5`.
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Reply:
Oh, my comment is wrong. I copied and pasted it from another one I wrote before. Let me modify it. Thank you!